lib/bootstrapper.js

```diff
@@ -0,0 +1,210 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: loopback-boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var assert = require('assert');
+var async = require('async');
+var utils = require('./utils');
+var path = require('path');
+var pluginLoader = require('./plugin-loader');
+var debug = require('debug')('loopback:boot:bootstrapper');
+var Promise = global.Promise || require('bluebird');
+
+module.exports = function(options) {
```

Comment:
I personally prefer to export `Bootstrapper` directly. 

```js
module.exports = Bootstrapper;
```

When I look at the code above, I see 2 places with `require('...boostrapper').Bootstrapper` and only one place calling the exported function directly.